frontend/lib/src/components/widgets/TextArea/TextArea.test.tsx

```diff
@@ -180,53 +168,29 @@ describe("TextArea widget", () => {
     )
   })
 
-  it("sets widget height if it is passed from props", () => {
```

Comment:
Tried to replicate this test but essentially its prop testing. See added `height` e2e test